USAGE.md

```diff
@@ -61,6 +61,7 @@ Allowed parameter names are:
 * `post_container_build`: after building container images
 * `pre_deploy`: before deploying EDPM
 * `post_deploy`: after deploying EDPM
+* `post_ctlplane_deploy`: after Control Plane deployment
```

Comment:
This is maybe a signal for us to split step 06-deploy-edpm in two, one for openstack ctrlplane and one for edpm.